specs/packagemanager/PackageManagement.md

```diff
@@ -854,6 +911,8 @@ namespace Microsoft.Windows.Management.Deployment
         EnsureReadyOptions();
 
         AddPackageOptions AddPackageOptions { get; };
+
+        Boolean RegisterNewerIfAvailable { get; };
```

Comment:
Get-only, or should be get/set?

Reply:
Yeah, copy/pasta. This should be read-write. The AddPackageOptions is read-only because it is a runtime class. The property is read-only, but you can mutate the returned AddPackageObjects object.

Reply:
Yes, thanks.

RECOMMEND: Support get+set